ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/ethernet-interfaces.yml

```
@@ -1758,3 +1761,108 @@ ethernet_interfaces:
     spanning_tree_portfast: edge
     lldp:
       ztp_vlan: 112
+
+  - name: Ethernet82
+    description: Switchport_tap_tool
+    switchport:
+      tap:
+        allowed_vlan: 25
+        default:
+          groups:
+            - g2
+            - g1
+            - g3
+          nexthop_groups:
+            - nexthop_g1
+            - nexthop_g3
+            - nexthop_g2
+          interfaces:
+            - port-channel10
+            - ethernet4
+        identity:
+          id: 3
+          inner_vlan: 5
+        mpls_pop_all: true
+        native_vlan: 10
+        truncation:
+          enabled: true
+          size: 150
+        mac_address:
+          destination: 01:00:00:00:00:00
+          source: 01:23:45:67:89:ab
+        encapsulation:
+          gre:
+            strip: true
+            destinations:
+              - destination: 1.1.1.1
+                source: 1.1.1.2
+                protocols:
+                  - protocol: "0x0000"
+                    strip: true
+                strip: true
+              - destination: 2.1.1.2
+                protocols:
+                  - protocol: "0x0010"
```

Let's check what happens without quotes to verify if it works or not (if not then we need to mention it in schema)
```suggestion
                  - protocol: 0x0010
```

---

Tested it. This must be in double quotes, otherwise it considers it integrer and gives type error